index.d.ts

```diff
@@ -98,8 +98,8 @@ export type WellKnownDirectory =
  * @returns Optional file handle to save in place.
  */
 export function fileSave(
-  /** To-be-saved blob */
-  blob: Blob,
+  /** To-be-saved blob or stream&mimeType object*/
+  blobOrStream: Blob | { readable: ReadableStream; type: string },
```

Comment:
Do we need the object? Could it just be `ReadableStream`, as the variable name suggests?

Reply:
@tomayac Yes the mime type is required here, there is no easy way to detect the mimeType of the data coming through the stream without adding new dependencies. The mime type is required to build the blob in legacy mode